library/java/src/io/envoyproxy/envoymobile/engine/EnvoyConfiguration.java

```diff
@@ -19,16 +21,20 @@
    * @param dnsFailureRefreshSecondsBase base rate in seconds to refresh DNS on failure.
    * @param dnsFailureRefreshSecondsMax  max rate in seconds to refresh DNS on failure.
    * @param statsFlushSeconds            interval at which to flush Envoy stats.
+   * @param appVersion                   the App Version of the App using this Envoy Client.
```

Comment:
nit: lowercase `app version`/`app ID`

Reply:
keeping it consistent with the rest of the names